variables.tf

```diff
@@ -1,5 +1,9 @@
-/* general */
-variable "hosts" {
+
+/* GENERAL */
+
+# number of machines to provision
+# export TF_VAR_node_count=5
+variable "node_count" {
```

Comment:
`node_count` is more descriptive than `hosts`. And it would be good to use it throughout the project, but I didn't want to make a wide sweeping change that requires more testing.